src/main/java/com/thebois/models/beings/Colony.java

```diff
@@ -130,8 +130,13 @@ private boolean isIdle(final IBeing being) {
     }
 
     @Override
-    public void add(final IItem item) {
-        inventory.add(item);
+    public boolean isFull() {
+        return false;
+    }
```

Comment:
Its nice to have this method in the interface, but here it becomes useless. Maybe we should have another interface for beings, like `IBeingInventory`? What do you think?

Reply:
What we discussed: Delegate to inventory to make it super clear that colony has nothing to do with inventory.